src/util/desktophelper.cpp

```diff
@@ -0,0 +1,169 @@
+
+#include "desktophelper.h"
+
+#include <QtDebug>
+#include <QString>
+#include <QDesktopServices>
+#include <QDir>
+#include <QUrl>
+#include <QProcess>
+
+#ifdef Q_OS_LINUX
+#include <QtDBus/QtDBus>
+#include <QFileInfo>
+#endif
+
+namespace {
+    const QString kSelectInFreedesktop = "fd";
```

Comment:
This indentation is strange -- we usually don't let namespaces cause an increase in the indent level. Please fix?